web/templates/stream_settings/confirm_stream_privacy_change_modal.hbs

```diff
@@ -1,5 +0,0 @@
-<div class="confirm-stream-privacy-modal">
-    <p class="confirm-stream-privacy-info">
```

Comment:
It's strange that these classes were unused. @Vector73 do you remember why you included these classes in 144c2c2f110605e2efee25b49c970c97d5271ee9? They seem to have been never used, in which case this PR's choice to drop the classes is a good one, but just wanted to check.